powerflex/helper/sdc_host_dpkg.go

```diff
@@ -84,7 +83,7 @@ func (r *SdcHostResource) CreateUbuntu(ctx context.Context, plan models.SdcHostM
 
 		// upload sw
 		scpProv := client.NewScpProvisioner(sshP)
-		pkgTarget := filepath.Join(dir, "emc-sdc-package.tar")
+		pkgTarget := dir + "/" + "emc-sdc-package.tar"
```

Comment:
@shekhar-j  Have you tested this on all platforms?

Also, have you run a security check after these changes?

Reply:
To take care of edge case
```suggestion
		pkgTarget := strings.TrimSuffix(dir, "/") + "/" + "emc-sdc-package.tar"
```

Reply:
Functional and Acceptance tests for this resource are passing. Security scans are ok.